.travis.yml

```
@@ -35,6 +35,12 @@ install:
   # Install extra requirements for running tests and building docs.
   - pip install -r requirements-dev.txt
   - conda list --show-channel-url
+  # Install specific commit of regro/cf-scripts (unreleased source code)
+  - pip install git+https://github.com/regro/cf-scripts@db87055b000e0ce9ea2fa5a82f55be0fb15613f2
+  - |
+    wget https://raw.githubusercontent.com/regro/cf-scripts/master/requirements/run
+    conda install -c conda-forge --file run -y
+    rm -rf run
```

`-r` is too aggressive here. Better not to use it when not necessary.
```suggestion
    rm -fv run
```